compiler/lib/src/main/scala/util/Error.scala

```diff
@@ -211,6 +215,8 @@ sealed trait Error {
         Error.print (Some(loc)) ("passive component may not have async input")
       case SemanticError.PassiveStateMachine(loc) =>
         Error.print (Some(loc)) ("passive component may not have a state machine instance")
+      case SemanticError.PortNumberAlreadyInUse(n) =>
+        Error.print (None) (s"port number $n is already in use")
```

Comment:
need to call out which port number is causing the issue and where it was already used in the other connection